mzLib/FlashLFQ/FlashLfqEngine.cs

```diff
@@ -471,6 +474,74 @@ private void QuantifyMs2IdentifiedPeptides(SpectraFileInfo fileInfo)
             _results.Peaks[fileInfo].AddRange(chromatographicPeaks.ToList());
         }
 
+        /// <summary>
+        /// Used by the match-between-runs algorithm to determine systematic retention time drifts between
+        /// chromatographic runs.
+        /// </summary>
+        private RetentionTimeCalibDataPoint[] GetRtCalSpline(SpectraFileInfo donor, SpectraFileInfo acceptor, MbrScorer scorer)
```

Comment:
The lack of coverage in these if else statements is a bit concerning as this will be the main method for LC elution drifts.

Reply:
This is all legacy code that was rearranged. Originally it was at line 960